src/app/course/course-question-snippet/course-question-snippet.component.html

```diff
@@ -1,5 +1,14 @@
 <ng-container *ngIf="uqjs">
     <div [class.tui-container]="eventId" [class.tui-container_adaptive]="eventId">
+        <a
+            [routerLink]="['..']"
+            icon="tuiIconArrowLeft"
+            size="s"
+            tuiButton
+            appearance="flat"
+        >
+            Back to List of Challenges
```

Comment:
This component is used in both "Challenges" and "Assignments". The name should be set according to where it's used. (i.e in the assignment it should be "Back to List of Assignment and Exams" or something similar)